.github/workflows/docker.yml

```
@@ -0,0 +1,71 @@
+name: Docker Images
+
+on:
+  push:
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-web:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

Future is now old man
```suggestion
        uses: actions/checkout@v4
```